terraform/aws/components/lab-results/r53_record_nhais.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_route53_record" "lab-results_lb_record" {
```

Comment:
rename the file, lab-results, not nhais

Reply:
File renamed in 4f73b521d1e81787f7740d758eb3e02443fb8667